api/spec/v1.yaml

```diff
@@ -1638,6 +1670,52 @@ components:
       description: |
         An event emitted by the consensus layer.
 
+    RoothashMessageType:
+      type: string
+      enum:
+        - staking.transfer
+        - staking.withdraw
+        - staking.add_escrow
+        - staking.reclaim_escrow
+        - registry.update_runtime
+        - governance.cast_vote
+        - governance.submit_proposal
+      example: *roothash_message_type_1
+
+    RoothashMessageList:
+      allOf:
+        - $ref: '#/components/schemas/List'
+        - type: object
+          required: [roothash_messages]
+          properties:
+            roothash_messages:
+              type: array
+              items:
+                allOf: [$ref: '#/components/schemas/RoothashMessage']
+
+    RoothashMessage:
+      type: object
+      required: [runtime, round, index]
+      properties:
```

Comment:
Each of the properties needs a `description` and ideally an `example`.

Reply:
adding :bicyclist: